src/ClientBase.php

```diff
@@ -78,6 +115,7 @@ protected function setCredentials(array $config = [])
 
         $this->username = $config['username'];
         $this->password = $config['password'];
+        $this->customerNumber = $config['customerNumber'];
```

Comment:
While the property is correctly named in `camelCase`, configuration array keys should be `snake_case`: `$config['customer_name']`.